analysis/freq_plot.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+
+codecounts_week = pd.read_csv(
+    filepath_or_buffer = "output/codecounts_week.csv"
+)
+codecounts_total = codecounts_week.sum()
+
+#################################### plots 1 ###################################
+def plotstyle(axesrow, axescol, title):
+    axs[axesrow,axescol].set_ylabel('Count per week')
+    axs[axesrow,axescol].xaxis.set_tick_params(labelrotation=70)
+    #axs[0,0].set_ylim(bottom=0) # might remove this in future depending on count fluctuation
+    axs[axesrow,axescol].grid(axis='y')
+    axs[axesrow,axescol].legend()
+    axs[axesrow,axescol].set_title(title, loc='left', y=1)
+    axs[axesrow,axescol].tick_params(labelbottom=True)
+    axs[axesrow,axescol].spines["left"].set_visible(False)
+    axs[axesrow,axescol].spines["right"].set_visible(False)
+    axs[axesrow,axescol].spines["top"].set_visible(False)
+    
+
+fig, axs = plt.subplots(2, 2, figsize=(15,12), sharey=False,  sharex=True)
+
+axs[0,0].plot(codecounts_week.index, codecounts_week["probable_covid"], marker='o', markersize=2, label=f"""Code for probable case, N={codecounts_total["probable_covid"]}""")
```

Comment:
nit: this is a gurt long line. Might be easier to read on github/smaller screens as 

```
axs[0, 0].plot(
    codecounts_week.index,
    codecounts_week["probable_covid"],
    marker="o",
    markersize=2,
    label=f"""Code for probable case, N={codecounts_total["probable_covid"]}""",
)
```

Reply:
Another style point - it might be clearer to iterate through the (sub)categories you defined in the df within counts.py rather than lots of repeated function calls as is here

Reply:
Thanks. I've iterated it now.